src/particles/feedback_CIC_gpu.cu

```diff
@@ -21,7 +25,8 @@
 namespace supernova {
   curandStateMRG32k3a_t*  randStates;
   part_int_t n_states;
-  Real t_buff, dt_buff;
+  Real *d_snr, snr_dt, t_sn_start, t_sn_end;
```

Comment:
We decided that we would use the `dev_` prefix for device pointers.

Reply:
I'll make the change.